setup.py

```diff
@@ -3,24 +3,24 @@
 from setuptools import setup, find_packages
 
 version = {}
-exec(open("frankenstein/__init__.py", "r").read(), version)
+exec(open("frank/__init__.py", "r").read(), version)
 version = version['__version__']
 
 setup(
-    name="frankenstein",
+    name="frank",
     version=version,
     packages=find_packages(),
-    author="Jeff Jennings",
+    author="Richard Booth, Jeff Jennings, Marco Tazzari",
     author_email="jmj51@ast.cam.ac.uk",
-    description="Frankenstein desc",
-    long_description=open('README.rst').read(),
+    description="Frankenstein, the flux reconstructor",
+    long_description=open('README.md').read(),
     install_requires=[line.rstrip() for line in open("requirements.txt", "r").readlines()],
     extras_require={
         'test' : ['pytest'],
-        'docs-build' : ['sphinx', 'sphinxcontrib-fulltoc'],
+        'docs-build' : ['sphinx', 'sphinxcontrib-fulltoc', 'sphinx_rtd_theme', 'nbsphinx'],
         },
-    license="tbd",
-    url="tbd",
+    license="GPLv3",
+    url="https://github.com/discsim/frank",
```

Comment:
Should this be the url of the package or the docs?

Reply:
The package